backend/entities/AccessRequestStatusType.cs

```diff
@@ -38,4 +38,15 @@ public AccessRequestStatusType(string id, string description) : base(id, descrip
         }
         #endregion
     }
+
+    public static class AccessRequestStatusTypes
```

Comment:
Generally bad practice to hide a **public** class in another's file.

Reply:
It is good practice to co-locate related functionality, such as a static constants class that is intended for use with the sibling class.